src/auth/user-import-builder.ts

```diff
@@ -190,239 +285,192 @@ export class UserImportBuilder {
   }
 
   /**
-   * Validates and populates the hashing options of the uploadAccount request.
+   * Validates and returns the hashing options of the uploadAccount request.
    * Throws an error whenever an invalid or missing options is detected.
+   * @param {UserImportOptions} options The UserImportOptions.
+   * @param {boolean} requiresHashOptions Whether to require hash options.
+   * @return {UploadAccountOptions} The populated UploadAccount options.
    */
-  private validateAndPopulateOptions(): void {
-    this.validatedOptions = {};
-    if (!this.requiresHashOptions) {
-      return;
+  private populateOptions(
+      options: UserImportOptions, requiresHashOptions: boolean): UploadAccountOptions {
+    let populatedOptions: UploadAccountOptions = {};
```

Comment:
Can this be just `let populatedOptions: UploadAccountOptions;`?

Reply:
I actually return this when hash options are required but that is fine, i will return `{}` directly.